utils/convertToWeiSafely.ts

```diff
@@ -0,0 +1,20 @@
+import { utils } from "ethers";
+const { parseUnits: toWei } = utils;
+
+// Minimum precision for a token or collateral currency that this dApp can support
+const MIN_TOKEN_PRECISION = 6;
```

Comment:
Throughout this PR I round all numbers before calling `toWei` on them because its possible that the string to convert `toWei` has more decimals than the number in `wei` can convert, which throws an error. Example, converting `toWei(0.123456789, 8)` will throw an error because there are 9 decimals but the precision specified is 8.

I hard code 6 as a safe precision value meaning this dApp will work for all tokens with >= 6 precision

Reply:
Wait, @nicholaspai why not just use the number of decimals to round? `.toFixed(decimals)`? (I could be missing some context here on why that would be hard/annoying)

Reply:
@mrice32 actually I think you can do that

Reply:
Fixed